corehq/apps/reports/filters/case_list.py

```diff
@@ -30,7 +30,8 @@ def static_options(self):
             ("all_data", _("[All Data]")),
             ('project_data', _("[Project Data]"))
         ]
-        filters_to_display.extend(self.create_static_option(option) for option in hq_user_type_options)
+        from corehq.apps.reports.filters.users import EmwfUtils
+        filters_to_display.extend(EmwfUtils.user_type_tuple(option) for option in hq_user_type_options)
```

Comment:
`EmwfUtils` is a superclass of `CaseListFilterUtils`, so you can reference `self.user_type_tuple` rather than importing the parent class.